en/brands.ftl

```diff
@@ -123,7 +123,7 @@
 -brand-name-mozilla-account = Mozilla account
 -brand-name-mozilla-accounts = Mozilla accounts
 -brand-name-mozilla-social = Mozilla.social
--brand-name-mozilla-ai = Mozilla AI
+-brand-name-mozilla-ai = Mozilla.ai
```

Comment:
Will these trigger updates in Pontoon? (I'm not sure whether brands/"terms" also need versioning, or all the strings using them will pick up the change nonetheless?)

Reply:
> Will these trigger updates in Pontoon? (I'm not sure whether brands/"terms" also need versioning, or all the strings using them will pick up the change nonetheless?)

Looks like most of the locales "have translated" [the term](https://transvision.mozfr.org/string/?entity=mozilla_org/en/brands.ftl:-brand-name-mozilla-ai&repo=mozilla_org) in the brands.ftl file. Typically, once the string is translated, changes in the source string won't trigger a replacement such as this.

Reply:
So what's the best approach to correct this name? Should I add the corrected name as a new string with `-v2` on the ID and mark the old one as obsolete?

Reply:
Either that, or I manually go through all the strings that are already "translated" and update them.

Reply:
@craigcook I will go ahead and merge this. If there is no simple way of fixing it, I will fix them one by one.

Reply:
I've versioned the string, and that will land shortly. No need to manually fix.

Reply:
That requires versioning and updating also all the strings referencing that term.

Reply:
yes, so far it's only in use in two places and we've updated them

Reply:
In case you decided to replace existing translations, there's a [branch to create a PR from](https://github.com/mozilla-l10n/www-l10n/compare/fix_mozai?expand=1) (but also heading offline).